### ISSUE TITLE
Cannot read property 'height' of undefined

### ISSUE THREAD

**Issue by nathanwoulfe** [jonyet/webdriverio-boilerplate#1]
Trying to run testcss or testcssremote throws a TypeError:

```
Error: Promise was rejected with the following reason: TypeError: Cannot read property 'height' of undefined
    at Object.wait (d:\webdriverio-boilerplate\node_modules\fibers\future.js:449:15)
    at Object.assertElementLayout (d:\webdriverio-boilerplate\node_modules\wdio-sync\build\index.js:345:27)
    at Context.<anonymous> (d:/webdriverio-boilerplate/test/specs_css/amazonvisualregression.js:16:13)
    at d:\webdriverio-boilerplate\node_modules\wdio-sync\build\index.js:649:20
    - - - - -
    at screenshot() - d:\webdriverio-boilerplate\node_modules\webdriverio\build\lib\commands\saveScreenshot.js:16:17
    at saveScreenshot() - d:\webdriverio-boilerplate\node_modules\wdio-sync\build\index.js:293:37
    at elementIdLocation("1") - d:\webdriverio-boilerplate\node_modules\webdriverio\build\lib\commands\getLocation.js:37:21
```

where the getLocation function is not finding the referred element, in this case the navbar.

Aside - I've looked a heap of different testing frameworks/boilerplate setups in the last week or so, this makes the most sense, so thanks!

**Comment by jonyet** [comment]
interesting, this occurred right after cloning and following the readme instructions, no other changes? also, thanks, i'm glad you find this readable

**Comment by nathanwoulfe** [comment]
Did a bit more digging yesterday, looks like it's actually an issue in ImageMagick, where a call to identify from within the imageeasy module isn't returning anything, which leads to the failed promise. I can call ImageMagick directly and it works fine, but from imageeasy identify fails. Nathan WoulfeWeb and media consultantnathanw.com.au | 0413 512 083On 16 Dec. 2016 1:15 am, jonathan terry <notifications@github.com> wrote:interesting, this occurred right after cloning and following the readme instructions, no other changes? also, thanks, i'm glad you find this readable

**Comment by jonyet** [comment]
the the loglevel to debug in the logger.js (if you haven't already), it should help with seeing what all is going on inside of the Resembler. typically i only get this height issue when the parentScreenshot that was grabbed only got a viewport screenshot instead of a full DOM screenshot, and then the framework tells it to crop out at a coordinate below the fold (leading to an empty `.png` with no height/width). don't recall it happening with anything like a navbar, since that would always be at the top of the document. this is a perfectly fixable problem, but shouldn't be happening immediately upon clone. i blew away my local copy and cloned again/tested it just successfully this morning, to make sure nothing obvious broke.

i'm on the gitter channel for webdriverio pretty much all the time, in case you want to have a chat (https://gitter.im/webdriverio/webdriverio)

**Comment by nathanwoulfe** [comment]
All sorted - hadn't installed the legacy bits, so didn't have `identify.` I'd updated imageeasy to call `magick identify` instead, but that was causing the error. Uninstalled, reinstalled with legacy, and reverted changes to imageeasy, now it's all sweet.

TLDR - my fault.
